client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/style.scss

```diff
@@ -282,6 +282,13 @@
 	}
 }
 
+// pending disabled tasks
+.launchpad__task.pending.disabled {
+	.launchpad__checklist-item-text {
+		color: inherit;
```

Comment:
This fixes the hover color on the disabled pending tasks (launch Link In Bio)

Reply:
Confirmed this is fixed.